addon/styles/addon.scss

```diff
@@ -1,38 +1,63 @@
 @import 'node_modules/ember-frost-css-core/scss/frost-core';
 @import 'node_modules/ember-frost-theme/scss/frost-theme';
 
-$border: 1px solid $frost-color-lgrey-5;
+$border: 1px solid $frost-color-grey-6;
```

Comment:
I'd like to see theme variables set up for these colors - even if it's just variables in this file for now